src/components/header/Header.tsx

```diff
@@ -63,42 +65,42 @@ const HeaderBlock = styled.header`
 `;
 
 const PreHeader = styled.div`
-  display: flex;
-  align-items: center;
-  justify-content: center;
   padding: 5px;
-  background-color: ${({ theme }) => theme.colors.red};
+  background-color: #ca3228;
   p {
+    text-align: center;
     color: ${({ theme }) => theme.colors.secondary};
-    font-size: ${({ theme }) => theme.fontSizes.xstext};
+    font-size: ${px2vw(40)};
+    @media ${device.lg} {
+      font-size: ${({ theme }) => theme.fontSizes.xstext};
+    }
   }
 `;
 
 const BlockLogo = styled.div`
   display: flex;
-  flex-direction: column;
   padding: 1rem;
   img {
-    width: 20vw;
-    height: auto;
+    width: auto;
+    height: 2.5vh;
+    &:last-child {
+      border-left: 1px solid ${({ theme }) => theme.colors.red};
+      padding-left: 0;
+      margin-left: ${px2vw(20)};
+    }
   }
 
   @media ${device.lg} {
-    flex-direction: row;
-    background-color: ${({ theme }) => theme.colors.secondary};
     img {
-      width: auto;
       height: 5vh;
-      &:last-child {
-        border-left: 1px solid ${({ theme }) => theme.colors.red};
-        padding-left: 0;
-        margin-left: 0.75rem;
-      }
     }
   }
 `;
 
 export default function Header() {
```

Comment:
Since this is kind of a new component and it's specific to your instance, you should probably add it to a `components/alimentaire/` directory already. This way it will be easier to identify all the instance specificity

_We will find_ a better way to way to handle instance specifics soon.

Reply:
Ok, so i must begin the refacto of the instance specifity.